dockerfile-image-update-itest/src/main/java/com/salesforce/dockerfileimageupdate/itest/TestStarter.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+package com.salesforce.dockerfileimageupdate.itest;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.*;
+import org.testng.xml.XmlSuite;
+
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by afalko, smulakala, minho.park on 7/14/16.
+ */
+public class TestStarter {
```

Comment:
Is there an advantage of using this rather than just running something like [Maven Failsafe](http://maven.apache.org/surefire/maven-failsafe-plugin/)

Reply:
We didn't want to exec this with maven...we run it standalone in docker-compose.

Reply:
Can't that kick off Maven?

Reply:
Perhaps, but experimenting with that would be for another PR